extensions/adobe/experience/adcloud/search/profile-adcloud.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/adcloud/search/profile-adcloud",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Ad Cloud Record",
+  "type": "object",
+  "auditable": false,
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/record"
+  ],
+  "description":
+    "An Adobe Ad Cloud Record is a custom class to represent and define Ad Cloud Entity Level Attributes.",
```

Comment:
This description should be improved. When should I use this class? What are `Ad Cloud Entity Level Attributes`?

Reply:
Description improved and made more meaningful across all other mix-in and data-types.